### ISSUE TITLE
ExoMol linelist parsing

### ISSUE THREAD

**Issue by ajwheeler** [ajwheeler/Korg.jl#279]
Requested by Nordlander.

**Comment by ajwheeler** [comment]
I'm going to summarize problems I'm having with this.  As a case study, I'm trying to take the [ExoMol TiO linelist](https://doi.org/10.1093/mnras/stz1818) and reproduce the tabulated cross-sections they provide for exoplanet atmospheric retrieval codes ([data here](https://www.exomol.com/data/data-types/linelist/TiO/48Ti-16O/Toto/)].

This plot compares (48)Ti(16)O cross-section at 3400K in a narrow wavelength range:
1) as calculated by ExoMol at $10^{-5}$ bar (TauREx format)
2) as calculated by ExoMol at $1$ bar (the rough pressure near the photosphere of an M dwarf) (TauREx format)
3) as calculated by Korg from the ExoMol linelist
4) as calculated by Korg from [Plez 1998](https://ui.adsabs.harvard.edu/abs/1998A%26A...337..495P/abstract) (this one is in a smaller wavelength range, because I only wanted to download one file from VALD)
![image](https://github.com/ajwheeler/Korg.jl/assets/711963/1205a643-0175-4fb9-a102-b1aacf1b8eff)

It's suspicious that my calculated cross-sections look similar, despite being from distinct linelists.

Here are some possible explanations:
- I've somehow got the ExoMol energies wrong (unlikely, because this and other bandheads are where they are supposed to be)
- I've somehow got the ExoMol oscillator strengths wrong (possible, but I've triple-checked everything and can't find a mistake)
- This has something to do with line broadening.  [ExoMol includes pressure broadening](https://www.aanda.org/10.1051/0004-6361/202038350). I'm not 100% clear on what what they are assuming about perturber density.  My calculations do not include pressure broadening, just estimated radiative and doppler broadening.  In any case, fiddling around with the profiles can't account for the big different in integrated intensity.
- Something to do with binning/resolution (see above).

Here's some key parameters of the strongest lines in this region:
![image](https://github.com/ajwheeler/Korg.jl/assets/711963/a80ab82b-03c9-4a62-9f02-151083eb3c99)
The strongest lines in the bands at the top are the source of nearly all the opacity.  It's suspicious that their lower-level degeneracies ($g$) grow with increasing wavelength (as does the difference between my and ExoMol's calculations).

**Comment by Trovemaster** [comment]
Dear Ajweeler, 

I have recomputed the TiO cross sections using the Doppler profile at T=3400 K and essentially reproduced your results:
![image](https://github.com/ajwheeler/Korg.jl/assets/12001159/f433b789-e5db-447c-afe1-0348a34af6be)

The difference with other two ExoMol simulations can be explained by a recent update (improvement) of the ToTo line list (but not cross sections and opacities), where we used experimental energies where available to replace the calculated onces. Your comparison clearly show we must update our opacities and cross secitons asap. We will do!

So, as far  as I can tell, you seem to use the line list correctly. 

Cheers, 
Sergey
